src/main/java/com/puppycrawl/tools/checkstyle/DefaultLogger.java

```diff
@@ -70,6 +70,12 @@ public class DefaultLogger extends AbstractAutomaticBean implements AuditListene
     /** Formatter for the log message. */
     private final AuditEventFormatter formatter;
 
+    /** Options for the info stream. */
+    private OutputStreamOptions infoStreamOptions;
+
+    /** Options for the error stream. */
+    private OutputStreamOptions errorStreamOptions;
+
```

Comment:
looks like this not required also, if we removing validatin

Reply:
dome